extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/StreamReader.java

```diff
@@ -14,14 +15,15 @@
      * Converts an InputStream to a BarrageMessage in the context of the provided parameters.
      *
      * @param options the options related to parsing this message
+     * @param expectedColumns the columns expected to appear in the stream, null implies all columns
```

Comment:
If and only if you have to edit this code again for some other reason, I would apply this minor cosmetic cleanup:
```
     * @param expectedColumns the columns expected to appear in the stream; null implies all columns
     * @param columnChunkTypes the types to use for each column chunk
     * @param columnTypes the actual type for the column
     * @param componentTypes the actual component type for the column
     * @param stream the input stream that holds the message to be parsed
     * @return a BarrageMessage filled out by the stream's payload
     */
    BarrageMessage safelyParseFrom(StreamReaderOptions options,
            BitSet expectedColumns,
            ChunkType[] columnChunkTypes,
            Class<?>[] columnTypes,
            Class<?>[] componentTypes,
            InputStream stream);
```